components/features.tsx

```diff
@@ -223,8 +235,23 @@ export default function Features() {
                             buttons={["play", "repeat", "frame", "debug"]}
                           />
                         </Player>
+                        <p className='text-left mt-4 font-bold text-secondary-400'>Try keploy Locally</p>
```

Comment:
Hey @sahadat-sk 
Please move this out of this div so that its position doesn't get affected by GIF and follow the color scheme from README for tooltip color

[Screencast from 22-03-24 06:25:53 PM IST.webm](https://github.com/keploy/website/assets/78696767/43b9d971-924b-4894-a8b2-9b3782a32a41)

Reply:
i have made change please have look